code_to_gast/helpers/py_helpers.py

```diff
@@ -176,6 +176,47 @@ def name_to_gast(node):
 def unary_op_to_gast(node):
     return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
 
+"""
+takes argument from function declaration and makes into gast node
+"""
+def arg_to_gast(node):
+    return pr.node_to_gast(node.arg)
+
+"""
+Takes native string class from python and creates gast node
+This node is used to store function names and arguments which
+is why the node type is name for identifier
+"""
+def str_to_gast(node):
+    return {"type": "name", "value": node}
+
+"""
+Turns return statement into gast
+"""
+def return_statement_to_gast(node):
+    return {"type": "returnStatement", "value": pr.node_to_gast(node.value)}
+
+"""
+Handles args of function declarations with default values
+"""
+def function_args_to_gast(node):
+    return arg_helper(node.args, node.defaults, [])
+
+def arg_helper(arg_list, default_list, param_list): 
```

Comment:
These helpers (which we have all been using) go against the original principles of relying on the router. Wonder if there's a way to incorporate them into our router in some other PR.